pkg/infrastructure/kogito_infra.go

```diff
@@ -36,3 +36,13 @@ func FetchKogitoInfraInstance(client *client.Client, name string, namespace stri
 		return instance, nil
 	}
 }
+
+// IsKafkaResource checks if provided KogitoInfra instance is for kafka resource
+func IsKafkaResource(instance *v1alpha1.KogitoInfra) bool {
```

Comment:
shouldn`t it be on the kafka class?

Reply:
Yes it should, I guess I lost this change when I rebased. Thanks!